src/components/molecules/EntityGrid/EntityGridItemPure.tsx

```diff
@@ -15,85 +15,99 @@ import Colors from '@styles/Colors';
 
 import {DetailsWrapper, ItemColumn, ItemRow, ItemWrapper, RowsWrapper, StyledMetricItem} from './EntityGrid.styled';
 
-const EntityGridItemPure: React.FC<any> = memo(forwardRef<HTMLDivElement, any>((props, ref) => {
-  const {item, onClick, entity, metrics} = props;
-  const {dataItem, latestExecution} = item;
+const EntityGridItemPure: React.FC<any> = memo(
+  forwardRef<HTMLDivElement, any>((props, ref) => {
+    const {item, onClick, entity, metrics, abortAllExecutions} = props;
+    const {dataItem, latestExecution} = item;
 
-  const status = latestExecution ? latestExecution?.executionResult?.status || latestExecution?.status : 'pending';
+    const status = latestExecution ? latestExecution?.executionResult?.status || latestExecution?.status : 'pending';
```

Comment:
nit: Is `pending` possible status? Based on [**back-end**](https://github.com/kubeshop/testkube/blob/9ff44abfb6eeb1521fe92c6d58849825e7a7a85e/pkg/api/v1/testkube/model_execution_status.go#L12):
```
const (
	QUEUED_ExecutionStatus  ExecutionStatus = "queued"
	RUNNING_ExecutionStatus ExecutionStatus = "running"
	PASSED_ExecutionStatus  ExecutionStatus = "passed"
	FAILED_ExecutionStatus  ExecutionStatus = "failed"
	ABORTED_ExecutionStatus ExecutionStatus = "aborted"
	TIMEOUT_ExecutionStatus ExecutionStatus = "timeout"
)
```

Reply:
Not entirely sure, it could be some old one

Reply:
Need to review these statuses, there are more not used by back end. But some are used, although not in this list